src/pages/spacewalk/transactions/index.tsx

```diff
@@ -108,7 +114,16 @@ function Transactions(): JSX.Element {
 
       return entries;
     };
-    fetchAllEntries().then((res) => setData(res));
+
+    setIsLoading(true);
```

Comment:
What do you think about using Tanstack react-query instead?

Reply:
Hmm right, we can do that.

Reply:
Once Implemented, the PR can be merged ✅

Reply:
Fixed that 👍